models/token.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    token: Joi
+        .string()
+        .token(),
+
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    userId: Joi
+        .number()
+        .integer()
+        .positive()
+        .description('User ID'),
+
+    name: Joi
```

Comment:
Secret names are used by the build systems, so I can understand why they follow certain conventions. Token names are for the human users to better differentiate between various secrets; the system won't care what they're named. Why would we specifically want token names to be [all uppercase and without spaces](https://github.com/screwdriver-cd/data-schema/blob/master/config/regex.js#L21)